cmd/oci-create-runtime-bundle/main.go

```diff
@@ -130,6 +130,10 @@ func (v *bundleCmd) Run(cmd *cobra.Command, args []string) {
 
 	case image.TypeImage:
 		err = image.CreateRuntimeBundle(args[0], args[1], v.ref, v.root)
+
+	default:
+		err = fmt.Errorf("cannot create %q", v.typ)
```

Comment:
Using `%q` gives you unambiguous quoting when the type contains strange characters:

```
$ cat test.go 
package main

import "fmt"

func main() {
        fmt.Printf("%q\n", "foo\"bar")
        fmt.Printf("%s\n", "foo\"bar")
        fmt.Printf("%q\n", "foo\nbar")
        fmt.Printf("%s\n", "foo\nbar")
}
$ go run test.go 
"foo\"bar"
foo"bar
"foo\nbar"
foo
bar
```

In this case, `v.typ` can be set directly by the user, so I think we should avoid assuming that it contains no strange characters.